### ISSUE TITLE
Applications that are Tiled crash after Monitor shut off

### ISSUE THREAD

**Issue by johnmain** [koekeishiya/yabai#1062]
On one of my monitors I usually have alacritty and Geany tiled. 
If I shut my monitors off, all the tiled Applications crash.
This seems to be when OS X sees that the monitor has shut down as all the other screens go black for a second.
The issue doesn't happen when Yabai service isn't running.

OS = Monterey
Steps:
Launch a couple apps on a monitor.
Shut off monitor
Turn monitor back on.
Note apps have crashed.

**Comment by joshtwo** [comment]
This may be why my iTerm2 and Slack instances crash when the displays sleep. Both of them are on an external monitor. Good observation!

**Comment by ahmed-rezk-dev** [comment]
Same here. the apps crash after my mac goes to sleep.

**Comment by NiBa97** [comment]
Same here

**Comment by zt1983811** [comment]
I got the same problem after upgrade to Monterey + yabai 4 ....

**Comment by peppy** [comment]
@zt1983811 Can you try disabling window borders if you are using that feature? It seemed to fix the issue for me. See https://github.com/koekeishiya/yabai/issues/1126 for a bit of related discussion.

**Comment by zt1983811** [comment]
> @zt1983811 Can you try disabling window borders if you are using that feature? It seemed to fix the issue for me. See #1126 for a bit of related discussion.

Thanks so much @peppy , it works like a charm !!! 
So for people who don't know please search border keyword in your `.yarbai.rc` file. put border off and disable the width and color for it as well. See below ⬇️ 
```
yabai -m config window_border               off
# yabai -m config window_border_width         2
# yabai -m config active_window_border_color  "0xE0${focused_border_color}"
# yabai -m config insert_window_border_color  "0xE0${preselect_border_color}"
# yabai -m config normal_window_border_color  "0x00${normal_border_color}"
```

**Comment by koekeishiya** [comment]
This should be resolved in v4.0.0.
